src/Definition/Redshift.php

```diff
@@ -58,9 +58,14 @@ public function getCompression()
     public function getSQLDefinition()
     {
         $definition =  $this->getType();
-        if ($this->getLength() && $this->getLength() != "") {
-            $definition .= "(" . $this->getLength() . ")";
+
+        $typesWithoutLength = ['TIMESTAMP', 'DATE', 'FLOAT', 'BOOLEAN'];
+        if (!in_array($this->getBasetype(), $typesWithoutLength)) {
```

Comment:
No, tohle ale není o tom, co vrací DB, ale co můžeš validně použít v definici datovýho typu. Pokud DB není schopná přijmout TIMESTAMP(0), tak je jedno, co vracej jakýkoliv metadata, ale tahle knihovna by TIMESTAMP(0) taky přijímat neměla. 

Pokud je to jakkoliv komplikovanější, tak by tuhle část měl řešit vlastní kód, kterej z metadat/information_schema vyextrahuje správný datový typy.